src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/TestRazorFormattingService.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.LanguageServer.Common;
+using Microsoft.AspNetCore.Razor.LanguageServer.Test;
+using Microsoft.AspNetCore.Razor.Test.Common;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting
+{
+    internal class TestRazorFormattingService
+    {
+        public static readonly RazorFormattingService Instance = CreateWithHtmlSupport(TestRazorCodeDocument.CreateEmpty());
+
+        private TestRazorFormattingService()
+        {
+        }
+
+        public static RazorFormattingService CreateWithHtmlSupport(RazorCodeDocument codeDocument)
```

Comment:
nit: Hmm, the name here implies to me that we're creating the formatting service with only HTML support rather than combined C#/HTML/Razor support. Would something like `CreateWithFullSupport` be appropriate instead?